pkg/api/customization/workload/aggregate_store.go

```diff
@@ -263,6 +268,31 @@ func setSecrets(apiContext *types.APIContext, data map[string]interface{}) {
 	}
 }
 
+func setScheduling(apiContext *types.APIContext, data map[string]interface{}) {
+	if _, ok := data["scheduling"]; ok {
```

Comment:
There's an API `values.GetValue(map, "key", "key", "key")` that may help here.  There also a corresponding `SetValue` method.

Reply:
Thank you, have updated using get and set value methods